src/main/java/org/mitre/synthea/engine/Generator.java

```diff
@@ -335,51 +344,293 @@ private synchronized void writeToConsole(Person person, int index, long time, bo
   }
 
   private long setDemographics(Person person, Demographics city) {
-    person.attributes.put(Person.CITY, city.city);
```

Comment:
Because the full SPEW dataset is too large to upload to github, I think we want to be able to choose to run either using SPEW or using the existing demographics format. At a glance I don't think this should be too hard, just have a config setting and choose between the 2

Reply:
Agreed.